library/bignum_core.c

```diff
@@ -33,39 +33,6 @@
 #include "bn_mul.h"
 #include "constant_time_internal.h"
 
```

Comment:
What could we put in our CI to ensure that if someone tries this again, the CI will complain?

Reply:
Yes, I'd like to know that too!